internal/dashboard/router/v1/obcluster_router.go

```diff
@@ -15,19 +15,20 @@ package v1
 import (
 	"github.com/gin-gonic/gin"
 
+	acbiz "github.com/oceanbase/ob-operator/internal/dashboard/business/ac"
 	h "github.com/oceanbase/ob-operator/internal/dashboard/handler"
 )
 
 func InitOBClusterRoutes(g *gin.RouterGroup) {
-	g.GET("/obclusters/statistic", h.Wrap(h.GetOBClusterStatistic))
-	g.GET("/obclusters", h.Wrap(h.ListOBClusters))
-	g.POST("/obclusters", h.Wrap(h.CreateOBCluster))
-	g.GET("/obclusters/namespace/:namespace/name/:name", h.Wrap(h.GetOBCluster))
-	g.POST("/obclusters/namespace/:namespace/name/:name", h.Wrap(h.UpgradeOBCluster))
-	g.DELETE("/obclusters/namespace/:namespace/name/:name", h.Wrap(h.DeleteOBCluster))
-	g.POST("/obclusters/namespace/:namespace/name/:name/obzones", h.Wrap(h.AddOBZone))
-	g.POST("/obclusters/namespace/:namespace/name/:name/obzones/:obzoneName/scale", h.Wrap(h.ScaleOBServer))
-	g.DELETE("/obclusters/namespace/:namespace/name/:name/obzones/:obzoneName", h.Wrap(h.DeleteOBZone))
-	g.GET("/obclusters/:namespace/:name/resource-usages", h.Wrap(h.ListOBClusterResources))
-	g.GET("/obclusters/:namespace/:name/related-events", h.Wrap(h.ListOBClusterRelatedEvents))
+	g.GET("/obclusters/statistic", h.Wrap(h.GetOBClusterStatistic, acbiz.PathGuard("obcluster", "*", "read")))
+	g.GET("/obclusters", h.Wrap(h.ListOBClusters, acbiz.PathGuard("obcluster", "", "read")))
```

Comment:
what's the difference between * and ""

Reply:
`*` means all of resource while empty string means any one of resource